apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.spec.ts

```diff
@@ -44,7 +45,7 @@ const mouseoverEvent = new MouseEvent('mouseover', {
     cancelable: true
 });
 
-xdescribe('DotEditContentHtmlService', () => {
+describe('DotEditContentHtmlService', () => {
```

Comment:
This test was failing intermittently :(

Reply:
mmm, this time it worked!
![image](https://user-images.githubusercontent.com/37185433/126233418-9ede10f5-fafd-4bd0-9355-1157fec6cc1b.png)

Reply:
2nd time after feedback!
![image](https://user-images.githubusercontent.com/37185433/126239866-f28e4795-404a-47d0-93ae-3f192f904358.png)

Reply:
3rd time after more feedback
![image](https://user-images.githubusercontent.com/37185433/126579355-cca7346c-5590-42c9-953d-18ccc42d0139.png)